ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/osdisk-custom-encryption/provision/cucushift-installer-rehearse-gcp-ipi-osdisk-custom-encryption-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/osdisk-custom-encryption/provision/cucushift-installer-rehearse-gcp-ipi-osdisk-custom-encryption-provision-chain.yaml",
```

Comment:
How about `gcp/ipi/disk/encryption/`, we might extend other disk feature integration.